src/error.rs

```diff
@@ -0,0 +1,88 @@
+//! Web3 Error
+
+#![allow(unknown_lints)]
+#![allow(missing_docs)]
+
+use std::io;
+use serde_json;
+use rpc;
+
+error_chain! {
```

Comment:
Hello

Thank you for the change :-). Though I might clarify that I asked for something slightly different ‒ I wanted your errors to implement the `std::error::Error` trait, which allows it to be used by a `foreign_link` inside `error_chain` on the user side (or, in general, to act as well-behaved error). But this works too (because `error_chain` implements the trait).

Reply:
Hey! Yeah, I know that `Error` trait would be enough, but error chain is awesome and we are migrating all our libraries to it anyways:)